lms/djangoapps/courseware/courses.py

```diff
@@ -618,7 +618,15 @@ def _get_default_child_module(child_modules):
             return _get_child(content_children) if content_children else None
 
     child = None
-    if hasattr(xmodule, 'position'):
+
+    try:
```

Comment:
@feanil 
```
In python 3, hasattr() catches AttributeErrors only then returns False. All other exceptions bubble up the call stack.
```

This error was appearing from [xblock lib](https://github.com/edx/XBlock/blob/master/xblock/fields.py#L518)